loader/loader.lua

```diff
@@ -1,83 +1,6 @@
 --- STEAMODDED CORE
 --- MODULE MODLOADER
 
--- Attempt to require nativefs
-local nfs_success, nativefs = pcall(require, "nativefs")
-local lovely_success, lovely = pcall(require, "lovely")
-
-local lovely_mod_dir
-local library_load_fail = false
-if lovely_success then
-    lovely_mod_dir = lovely.mod_dir:gsub("/$", "")
-else
-    sendErrorMessage("Error loading lovely library!", 'Loader')
-    library_load_fail = true
-end
-if nfs_success then
-    NFS = nativefs
-    -- make lovely_mod_dir an absolute path.
-    -- respects symlink/.. combos
-    NFS.setWorkingDirectory(lovely_mod_dir)
-    lovely_mod_dir = NFS.getWorkingDirectory()
-    -- make sure NFS behaves the same as love.filesystem
-    NFS.setWorkingDirectory(love.filesystem.getSaveDirectory())
-else
```

Comment:
Concerned about removing the block above.

Reply:
The relevant code wasn't removed, but relocated to `core.lua` because it needs to run earlier